circuit_maintenance_parser/providers.py

```diff
@@ -125,6 +125,13 @@ class PacketFabric(GenericProvider):
     _default_organizer = "support@packetfabric.com"
 
 
+class Telia(GenericProvider):
+    """Telia provider custom class."""
+
+    _parser_classes: Iterable[Type[Parser]] = [ICal]
+    _default_organizer = "ncm@teliacompany.com"
```

Comment:
Is there a reason the default organizer here doesn't match the ORGANIZER in the unit test?

Reply:
I believe it's because if the notification contains a organizer, the default is not used